Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackSystemComponent.h

```diff
@@ -90,6 +74,9 @@ namespace AZ
 
             //! Used for loading the pass templates.
             RPI::PassSystemInterface::OnReadyLoadTemplatesEvent::Handler m_loadTemplatesHandler;
+
+            //! Map of focused entity
+            AZStd::unordered_map<EntityId, FocuseMeshdEntity> m_focusedEntities;
```

Comment:
The focused mesh entity isn't a lightweight object (containing a vector) and it might be better to store them as unique_ptrs to avoid lots of data copies and allocations when modifying the map.

Reply:
It looks like there's also a typo here `FocuseMeshdEntity` - I think this should be `FocusedMeshEntity` right?

Reply:
Correct, that's a typo that has been fixed. The class itself is a lightweight proxy for the actual entity's mesh data 👍